### ISSUE TITLE
Deletion of drafts

### ISSUE THREAD

**Issue by rghv** [hasgeek/hasjob#337]
There doesn't seem to be an option to delete a draft. Would be a useful feature?

**Comment by jace** [comment]
First time in 5+ years this feature request has come up, although it seems so obvious in hindsight.

**Comment by rghv** [comment]
Yup. Very easy to end up littering one's workspace if many drafts were left dangling without a way to delete them. Not sure if drafts automatically expire on HasJob after a certain time, like the job posts.

**Comment by iambibhas** [comment]
Introduce a `DELETED` state and hide it from everywhere.
